users/vieting/experiments/switchboard/ctc/feat/experiments.py

```diff
@@ -487,6 +780,88 @@ def run_scf_specaug_sort():
     return report
 
 
+def run_scf_baseline_decaying_batchsize():
+    gs.ALIAS_AND_OUTPUT_SUBDIR = "experiments/switchboard/ctc/feat/"
+
+    _, nn_system = run_scf_baseline()
+
+    (
+        returnn_datasets,
+        rasr_loss_corpus_path,
+        rasr_loss_corpus_segments,
+        rasr_loss_lexicon_path,
+        dev_corpora,
+    ) = get_datasets()
+    returnn_args = {
+        "batch_size": 5000,
+        "rasr_binary_path": RASR_BINARY_PATH,
+        "rasr_loss_corpus_path": rasr_loss_corpus_path,
+        "rasr_loss_corpus_segments": rasr_loss_corpus_segments,
+        "rasr_loss_lexicon_path": rasr_loss_lexicon_path,
+        "datasets": returnn_datasets,
+        "extra_args": {
+            "conv_pad_seq_len_to_power": 1.5,
+            "preload_from_files": {
+                "existing-model": {
+                    "filename": nn_system.train_jobs["conformer_bs2x5k_scf_baseline_preemphasis97_wn"].out_checkpoints[
+                        24
+                    ],
+                    "init_for_train": True,
+                }
+            },
+        },
+        "conformer_type": "wei",
+        "specaug_old": {"max_feature": 15},
+    }
+    feature_args = {
+        "class": "ScfNetwork",
+        "size_tf": 256 // 2,
+        "stride_tf": 10 // 2,
+        "preemphasis": 0.97,
+        "wave_norm": True,
+    }
+
+    nn_args, report_args_collection = get_nn_args_baseline(
+        nn_base_args={
+            "scf_baseline": dict(
+                returnn_args=returnn_args,
+                feature_args=feature_args,
+                lr_args={
+                    "peak_lr": 4e-4,
+                    "start_lr": 6.2668056e-05,
+                    "end_lr": 1e-5,
+                    "increase_epochs": 156,
+                    "decrease_epochs": 180,
+                    "final_epochs": 0,
+                },
+                report_args={
+                    "deacy_epoch": 24,
+                    "start_batch_size": 5000,
+                    "end_batch_size": 10000,
```

Comment:
Did you ignore my comments on this?
```suggestion
                    "batch_size_decay_epoch": 24,
                    "batch_size_start": 5000,
                    "batch_size_end": 10000,
```

Reply:
Max, you repeatedly mark my comments as resolved without resolving them. This is really annoying!

Reply:
fixed it now. change was reverted back earlier by a force push